jberet-schedule-executor/pom.xml

```diff
@@ -78,7 +37,8 @@
         </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.module</groupId>
-            <artifactId>jackson-module-jaxb-annotations</artifactId>
+            <artifactId>jackson-module-jakarta-xmlbind-annotations</artifactId>
+            <version>${version.com.fasterxml.jackson}</version>
             <scope>test</scope>
         </dependency>
```

Comment:
any reason for removing these test dependencies? I can see ejb-api has been moved to parent pom, which is fine. But other test deps are needed to run tests in `org.jberet.schedule.ExecutorSchedulerIT`. This test is currently ignored to save build time, so the default mvn build will not run it. I think we should probably remove the @Ignore and tolerate the somewhat long build time.

Reply:
You are right, I removed and Ignore annotation, the test passes.

Reply:
Of course I also restored the test dependencies for the test.